.github/workflows/postman-collection-runner.yml

```diff
@@ -3,8 +3,8 @@ name: Run postman tests
 on:
   workflow_dispatch:
   pull_request:
```

Comment:
Do we need the PR trigger either?

Reply:
It will just skip the tests and hence it will pass no matter what